cli/tests/activate.bats

```diff
@@ -232,10 +312,30 @@ env_is_activated() {
 
 # ---------------------------------------------------------------------------- #
 
-# bats test_tags=activate,activate:path
-@test "'flox activate' modifies path" {
+# bats test_tags=activate,activate:path,activate:path:bash
+@test "'flox activate' modifies path (bash)" {
+  original_path="$PATH"
+  run "$FLOX_BIN" activate -- bash -c 'echo $PATH'
+  assert_success
+  assert_not_equal "$original_path" "$output"
+
+  # hello is not on the path
+  run -1 type hello
+
+  run "$FLOX_BIN" install hello
+  assert_success
+
+  FLOX_SHELL="$(which bash)" run "$FLOX_BIN" activate --dir "$PROJECT_DIR" -- hello
+  assert_success
+  assert_output --partial "Hello, world!"
+}
+
+# ---------------------------------------------------------------------------- #
+
+# bats test_tags=activate,activate:path,activate:path:zsh
+@test "'flox activate' modifies path (zsh)" {
   original_path="$PATH"
-  run "$FLOX_BIN" activate -- echo '$PATH'
+  run "$FLOX_BIN" activate -- bash -c 'echo $PATH'
```

Comment:
No, because invoking zsh outside of a `flox activate` will source system and user config files that can reset PATH altogether. That said, what we really want here is to report `$PATH` without invoking a subshell at all, and we can do that by reverting that part of the patch altogether. Good catch - fixed in 89ac26df406625f9ca95770236d040713f14f20d